src/Explorer/Panes/CassandraAddCollectionPane.ts

```diff
@@ -538,17 +513,12 @@ export default class CassandraAddCollectionPane extends ContextualPaneBase {
     ) {
       const autoPilot = this._getAutoPilot();
       if (
-        (!this.hasAutoPilotV2FeatureFlag() &&
-          (!autoPilot ||
-            !autoPilot.maxThroughput ||
-            !AutoPilotUtils.isValidAutoPilotThroughput(autoPilot.maxThroughput))) ||
-        (this.hasAutoPilotV2FeatureFlag() &&
-          (!autoPilot || !autoPilot.autopilotTier || !AutoPilotUtils.isValidAutoPilotTier(autoPilot.autopilotTier)))
+        !autoPilot ||
+        !autoPilot.maxThroughput ||
+        !AutoPilotUtils.isValidAutoPilotThroughput(autoPilot.maxThroughput)
       ) {
         this.formErrors(
-          !this.hasAutoPilotV2FeatureFlag()
-            ? `Please enter a value greater than ${AutoPilotUtils.minAutoPilotThroughput} for autopilot throughput`
-            : "Please select an Autopilot tier from the list."
+          `Please enter a value greater than ${AutoPilotUtils.minAutoPilotThroughput} for autopilot throughput`
```

Comment:
It's interesting that the `AddDatabasePane` and `AddCollectionPane` have duplicate code.

Reply:
Agreed. Doing this was mostly a find and replace exercise, but it def made me realize there are other issues with all this code...